### PR TITLE
some last fixes before first beta

### PR DESCRIPTION
**Pull request by mkz212**
- more logs info
- some cleaning
- added ecoFunctionData

### CODE REVIEW

File: src/accessories/indoor-unit.ts [hunk 1]
```diff
@@ -143,18 +143,16 @@ export default class IndoorUnitAccessory {
       // Temperature of 126 from the API = null/failure
       
       if (deviceStatus.insideTemperature < 126) {
-        this.service.updateCharacteristic(this.platform.Characteristic.CurrentTemperature, deviceStatus.insideTemperature);
-      }
-      
-      else {
+	      this.service.updateCharacteristic(this.platform.Characteristic.CurrentTemperature, deviceStatus.insideTemperature);
+	      this.platform.log.info(`Indoor temperature: '${deviceStatus.insideTemperature}'`);
```
Comment (embee8) [inline src/accessories/indoor-unit.ts:147]: Why log this as info? Do we have a sense of how large the log file will grow after a few months of operation on an average installation? I am aware of situations where space has been eaten up by it, eventually leading to a crash of the machine Homebridge is running on.

Comment (mkz212) [reply src/accessories/indoor-unit.ts:147]: Right attention! I will change it to debug.

**Comment by embee8** [comment]
There's quite a few hidden linter errors that are not surfaced in the checks because there has been an issue with linting files in certain directories. This has been fixed in #98.

I am going to merge the changes onto your feature branch, but given the new release workflow, I'd like to ask you to merge the feature branch with the master branch in your repo, correct the linter errors, and submit a PR to the upstream repository.

Once all that is done, we'll delete the feature branch and proceed as outlined in #94.

**Comment by embee8** [comment]
Tip for next time: Please use more descriptive commit messages, not just "Update file".

**Pull request merged**